docs/resources/index.md

```diff
@@ -82,6 +95,20 @@ Optional:
 - `patch_method` (String)
 
 
+<a id="nestedatt--settings--model_properties"></a>
+### Nested Schema for `settings.model_properties`
```

Comment:
This feels like a duplication; can they both refer to the same schema?

Reply:
Currently not, the schemas in datasource are all string/bool, whereas there are some Number types in the resource. I tried having datasource refer to the schemas with Number types before but this gives an error.